### PR TITLE
espeak-ng: update to 1.51

### PR DESCRIPTION
**Pull request by CMB**
- pcaudiolib: update to 1.2.
- espeak-ng: update to 1.51.

- I tested the changes in this PR: **YES**

- I built this PR locally for my native architecture, (x86_64-musl)
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - aarch64-musl

### CODE REVIEW

File: srcpkgs/espeak-ng/template [hunk 1]
```diff
@@ -11,7 +11,7 @@ maintainer="Joey <joey@imap.cc>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/espeak-ng/espeak-ng/"
```
Comment (classabbyamp) [inline srcpkgs/espeak-ng/template:12]:
```suggestion
homepage="https://github.com/espeak-ng/espeak-ng/"
changelog="https://raw.githubusercontent.com/espeak-ng/espeak-ng/master/ChangeLog.md"
```

File: srcpkgs/pcaudiolib/template [hunk 2]
```diff
@@ -10,7 +10,7 @@ maintainer="Joey <joey@imap.cc>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/espeak-ng/pcaudiolib"
```
Comment (classabbyamp) [inline srcpkgs/pcaudiolib/template:11]:
```suggestion
homepage="https://github.com/espeak-ng/pcaudiolib"
changelog="https://raw.githubusercontent.com/espeak-ng/pcaudiolib/master/ChangeLog.md"
```

**Comment by CMB** [comment]
Done, thanks.

File: srcpkgs/espeak-ng/template [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'espeak-ng'
 pkgname=espeak-ng
-version=1.50
+version=1.51
 revision=1
 build_style=gnu-configure
 make_build_args="-j1"
```
Comment (paper42) [inline srcpkgs/espeak-ng/template:6]: We should use `disable_parallel_build=yes` instead

**Pull request merged**